### DISCUSSION TITLE
feature request -  rshell

### DISCUSSION THREAD

**Discussion by SpudGunMan** [pybricks/support#609] [Ideas]
adding [rshell](https://github.com/dhylands/rshell) project to standardize with other micropython projects common I/o

**Comment by SpudGunMan** [comment]
more specifically advertising a bluetooth or USB modem for connection to the existing shell would be nice

**Comment by laurensvalk** [comment]
There's also https://docs.micropython.org/en/latest/reference/mpremote.html

Both require file system access though, which isn't enabled yet.
